test/sub.mk

```diff
@@ -35,6 +38,12 @@ all:
 	if test $(COMPILER_JAR) -nt compile.stamp; then rm -f compile.stamp; fi
 	$(MAKE) -f ../../sub.mk tdir=$(tdir) compile
 
+errorOutput:$(err_logs)
```

Comment:
This all seems far too complicated. Just run the compiler once on all the `-e` files. Fix `main` to keep going when it gets an error. Not sure why you are doing this in `sub.mk` rather than the containing Makefile.

Reply:
fixed